highgui/src/capture_v4l2_rk_aiq.cpp

```diff
@@ -61,6 +61,12 @@ static int get_device_model()
             // luckfox pico family and plus pro max mini variants
             device_model = 1;
         }
+
+        if (strncmp(buf, "LockzhinerVisionModule", 12) == 0)
```

Comment:
```suggestion
        if (strncmp(buf, "LockzhinerVisionModule", 22) == 0)
```
`LockzhinerVisionModule` 有 22 个字符.

上面的例子, `Luckfox Pico` 有12个字符，所以用了12.
不能直接 copy 的.

Reply:
用 `sizeof` 代替了一下